testcontainers/compose.py

```diff
@@ -80,6 +83,16 @@ def stop(self):
             subprocess.call(["docker-compose", "-f", self.compose_file_name, "down", "-v"],
                             cwd=self.filepath)
 
+    def get_logs(self):
+        with blindspin.spinner():
+            result = subprocess.run(
+                ["docker-compose", "-f", self.compose_file_name, "logs"],
+                cwd=self.filepath,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
```

Comment:
```suggestion
                cwd=self.filepath, capture_output=True
```

Reply:
@tillahoffmann This does not work as `capture_output` was only added in 3.7.  See this commit https://github.com/testcontainers/testcontainers-python/pull/93/commits/7fe7631a108e0e1e678bf6ceb74a4766a9c1e8e6 and failed tests on it.

Reply:
Version that works in all supported version of python is in https://github.com/testcontainers/testcontainers-python/pull/93/commits/8a305e4af576a26f5b035e9937c09726f789262e

Reply:
Duh, sorry, let me revert that. 🤦